examples/cert-val.diag

```diff
@@ -0,0 +1,23 @@
+  {
```

Comment:
we need to have CDDL for this to validate against.

Reply:
Introduced a CDDL, kindly check!